composer.json

```diff
@@ -4,7 +4,7 @@
     "type": "library",
     "require": {
         "php": "^7.3|^8.0",
```

Comment:
```suggestion
        "php": "^8.0",
```

Reply:
Then 2.0 ?
As you can't use it on older NC versions anymore?